src/Process/3dTilesProcessing.js

```diff
@@ -96,7 +96,6 @@ export function pre3dTilesUpdate(context, layer) {
 function computeNodeSSE(camera, node) {
     if (node.boundingVolume.region) {
         const worldCoordinateCenter = node.boundingVolume.region.centerWorld;
-        worldCoordinateCenter.applyMatrix4(node.matrixWorld);
```

Comment:
And to transform in tile local coords I think you'd only need to apply the inverse of the transform defined in `tileset.json`...